### PR TITLE
Disable CORS

### PR DESCRIPTION
**Pull request by TiagoGraf89**
Enable CORS for development only

### CODE REVIEW

**Comment by healthgatewaybc** [comment]
Patient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=patientservice_pr_248)

**Comment by healthgatewaybc** [comment]
Immunization Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=immunization_pr_248)

**Comment by healthgatewaybc** [comment]
HNClient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=hnclientservice_pr_248)

**Comment by healthgatewaybc** [comment]
WebClient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=webclient_pr_248)

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_248)

**Comment by healthgatewaybc** [comment]
Immunization Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=immunization_pr_248)

**Comment by healthgatewaybc** [comment]
Patient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=patientservice_pr_248)

File: Apps/Common/src/AspNetConfiguration/StartupConfiguration.cs [hunk 1]
```diff
@@ -328,13 +328,17 @@ public void UseHttp(IApplicationBuilder app)
             app.UseHealthChecks("/health");
 
             // Enable CORS
-            app.UseCors(builder =>
+            string enableCors = this.configuration.GetValue<string>("AllowOrigins", "");
```
Comment (sslaws) [inline Apps/Common/src/AspNetConfiguration/StartupConfiguration.cs]: nit: "" should be string.Empty

**Comment by healthgatewaybc** [comment]
WebClient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=webclient_pr_248)

File: Apps/Common/src/AspNetConfiguration/StartupConfiguration.cs (hunk 1 above)
Comment (TiagoGraf89) [reply Apps/Common/src/AspNetConfiguration/StartupConfiguration.cs]: Done

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_248)

**Comment by healthgatewaybc** [comment]
Patient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=patientservice_pr_248)

**Comment by healthgatewaybc** [comment]
Immunization Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=immunization_pr_248)

**Comment by healthgatewaybc** [comment]
HNClient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=hnclientservice_pr_248)

**Comment by healthgatewaybc** [comment]
WebClient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=webclient_pr_248)

**Pull request merged**